cassandra/cqlengine/models.py

```diff
@@ -492,7 +494,10 @@ def column_family_name(cls, include_keyspace=True):
     def _raw_column_family_name(cls):
         if not cls._table_name:
             if cls.__table_name__:
-                cls._table_name = cls.__table_name__.lower()
+                if cls.__table_name_case_sensitive__:
+                    cls._table_name = cls.__table_name__
+                else:
+                    cls._table_name = cls.__table_name__.lower()
             else:
```

Comment:
Just brainstorming -- what would you think of trying to move away from transformed table names in the future?
Instead of transforming names from camelcase to '\_' delimited, we would always use the names directly. We could transition as follows:
1.) now, introduce a model attribute "transform table name" (or whatever you want to call it), defaulted ``True`` to match current behavior.
2.) Warn if transformed name != literal name. This will encourage people to change their model names to match the actual table name in the database
3.) Next major rev, change default "transform table name" to ``False``, and remove transform logic -- tables will always be named exactly ``cls.__table_name__`` or ``cls.__name__``.

lmkwyt